Solutions/AristaAwakeSecurity/Analytic Rules/HighSeverityMatchesByDevice.yaml

```diff
@@ -0,0 +1,62 @@
+id: d5e012c2-29ba-4a02-a813-37b928aafe2d
+name: Awake Security - High Severity Matches By Device
+description: This query searches for devices with high severity event(s).
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AristaAwakeSecurity
+    dataTypes:
+      - AwakeSecurity
+query: |
+  CommonSecurityLog
+  | where DeviceVendor == "Arista Networks" and DeviceProduct == "Awake Security" and toint(LogSeverity) > 6
+  | summarize Models=make_set(Activity), ASPMatchURLs=make_set(DeviceCustomString2), SourceIPs=make_set(SourceIP),
+    DestinationIPs=make_set(DestinationIP), ModelMatchCount=sum(EventCount), MaxSeverity=max(todecimal  (LogSeverity)) by SourceHostName
+queryFrequency: 1h
```

Comment:
The structure of the template should be fixed. queryFrequency, queryPeriod, triggerOperator, triggerThreshold, tactics, relevantTechniques should ideally come before the query.

Reply:
Fixed the structure for analytic rules